common/src/main/res/values/strings.xml

```diff
@@ -734,6 +734,7 @@
     <string name="sensor_setting_nextalarm_allow_list_title">Allow list</string>
     <string name="sensor_setting_notification_allow_list_title">Allow list</string>
     <string name="sensor_setting_notification_disable_allow_list_title">Disable allow list requirement</string>
+    <string name="sensor_setting_active_notification_count_content_attrs_title">Include content from all notifications as sensor attributes</string>
```

Comment:
Now rereading it, I think the word `sensor` is unnecessary here. WDYT?

```suggestion
    <string name="sensor_setting_active_notification_count_content_attrs_title">Include content from all notifications as attributes</string>
```

Reply:
I think that makes sense :)